Ambit/Config/FilterPlugin.ini

```diff
@@ -6,4 +6,5 @@
 ;    /README.txt
 ;    /Extras/...
 ;    /Binaries/ThirdParty/*.dll
-AWSAmbit.hda
\ No newline at end of file
+AWSAmbit.hda
+/Config/*
```

Comment:
Found some examples in the github: https://github.com/Phyronnaz/VoxelPlugin/blob/master/Config/FilterPlugin.ini
https://github.com/nfrechette/acl-ue4-plugin/blob/develop/ACLPlugin/Config/FilterPlugin.ini
These two are all published on the Marketplace but their `FilterPlugin.ini`s look totally different. I was confused by the guideline of the Epic Games now.

Reply:
I'm going to add some of the legal documents to the sub-directory, and then have them included in the distribution. `Content`, `Resources`, `Source` are all included by default, and are not necessary for the FilterPlugin to contain.